packages/client/src/components/SafeSettings.js

```diff
@@ -447,11 +447,11 @@ function SafeSettings({ address, web3, name, threshold, safeOwners }) {
     const thresholdToPersist = newThreshold ?? threshold;
 
     if (newOwner) {
-      await proposeAddSigner(formatAddress(newOwner.address));
+      await proposeAddSigner(web3, formatAddress(newOwner.address));
```

Comment:
can we access `web3` like in @linxiao-li 's PR: https://github.com/DapperCollectives/VESSEL/pull/52/files#diff-55a64a22c46f532689008bceccb29906ccc8380b87ccd064ec508835cad48958R12

Reply:
Moved createSignature function from util to web3 context